keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -585,34 +577,82 @@ instead. Obsolete objects need to be removed.
 
 To ensure that `CSIStorageCapacity` objects get removed when the driver
 deployment gets removed before it has a chance to clean up, each
-`CSIStorageCapacity` object needs an [owner
+`CSIStorageCapacity` object should have an [owner
 reference](https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#OwnerReference).
 
-For central provisioning, that has to be the Deployment or StatefulSet
-that defines the provisioner pods. That way, provisioning can
-continue seamlessly when there are multiple instances with leadership
-election.
-
-For deployment with a daemon set, making the individual pod the owner
-is better than the daemon set as owner because then other instances do
-not need to figure out when to remove `CSIStorageCapacity` objects
-created on another node. It is also better than the node as owner
-because a driver might no longer be needed on a node although the node
-continues to exist (for example, when deploying a driver to nodes is
-controlled via node labels).
-
-While external-provisioner runs, it needs to update and potentially
+The owner should be the higher-level app object which defines the provisioner pods:
+- the Deployment or StatefulSet for central provisioning
+- the DaemonSet for distributed provisioning
+
+This way, provisioning and pod scheduling can continue seamlessly when
+there are multiple instances with leadership election or the driver
+gets upgraded.
+
+This ownership reference is optional. It gets enabled with
+external-provisioner command line arguments when deploying the
+driver. Without it, administrators must manually delete
+`CSIStorageCapacity` objects after uninstalling a CSI driver. To
+simplify that and for efficient access to objects, external-provisioner
+sets some labels:
+- `app.kubernetes.io/instance`: the CSI driver name
+- `app.kubernetes.io/managed-by`: `external-provisioner` for central provisioning,
+  `external-provisioner-<node name>` for distributed provisioning
+
+With that, `kubectl delete csistoragecapacities -l
+app.kubernetes.io/instance=my-csi.example.com` will delete just the
+objects of that driver.
+
+It is possible to create `CSIStorageCapacity` manually. This may be
+useful to specify capacity of simulated nodes before the autoscaler
```

Comment:
note autoscaling simluations are all in memory.

Reply:
Yes, but for simulated nodes, it only creates objects in memory.

Reply:
Since this isn't accurate of how autoscaler works, I would remove the autoscaler part, say that it shouldn't be done, and describe the behavior if it does happen.

Reply:
Done.